vue/components/ui/molecules/table/table.vue

```diff
@@ -82,16 +82,25 @@
     white-space: pre;
 }
 
+.table.dense th {
+    font-weight: 600;
+}
+
 .table ::v-deep td {
     font-size: 14px;
     font-weight: 600;
     height: 80px;
     overflow: hidden;
-    padding: 0px 0px 0px 0px;
+    padding: 0px 20px 0px 20px;

```

Comment:
Why?

Reply:
When aligned left or right, the text in the body of the table isn't aligned with the table header. This was a way of styling independently of the text-alignment being made.